docs/core/releases-and-support.md

```diff
@@ -59,29 +59,28 @@ There are two support tracks for releases:
 
 * *Standard Term Support* (STS) releases
 
-  These versions are supported until 6 months after the next major or minor release ships.
+  These versions are supported for 2 years (24 months).
 
   Example:
 
-  * .NET 5 is an STS release and was released in November 2020. It was supported for 18 months, until May 2022.
-  * .NET 7 is an STS release and was released in November 2022. It's supported for 18 months, until May 2024.
+  * .NET 9 is an STS release and was released in November 2024. It's supported for 2 years, until November 2026.
 
 * *Long Term Support* (LTS) releases
 
   These versions are supported for a minimum of 3 years, or 1 year after the next LTS release ships if that date is later.
 
-  Example:
+  Examples:
 
   * .NET Core 3.1 is an LTS release and was released in December 2019. It was supported for 3 years, until December 2022.
-  * .NET 6 is an LTS release and was released in November, 2021. It's supported for 3 years, until November 2024.
+  * .NET 8 is an LTS release and was released in November, 2023. It's supported for 3 years, until November 2026.
 
-Releases alternate between LTS and STS, so it's possible for an earlier release to be supported longer than a later release. For example, .NET Core 3.1 was an LTS release with support through December 2022. The .NET 5 release shipped almost a year later but went out of support earlier, in May 2022.
+Releases alternate between LTS and STS.
 
 Servicing updates ship monthly and include both security and non-security (reliability, compatibility, and stability) fixes. Servicing updates are supported until the next servicing update is released. Servicing updates have runtime roll forward behavior. That means that applications default to running on the latest installed runtime servicing update.
 
 ## How to choose a release
 
-If you're building a service and expect to continue updating it on a regular basis, then an STS release like the .NET 7 runtime may be your best option to stay up to date with the latest features .NET has to offer.
+If you're building a service and expect to continue updating it on a regular basis, then an STS release like the .NET 7 runtime might be your best option to stay up to date with the latest features .NET has to offer.
```

Comment:
```suggestion
If you're building a service and expect to continue updating it on a regular basis, then an STS release like the .NET 9 runtime might be your best option to stay up to date with the latest features .NET has to offer.
```

Reply:
this paragraph is a bit misleading though. you should always pick the latest (STS or LTS) if you're updating on a regular basis